src/helper/markdown/lineQueries.ts

```diff
@@ -0,0 +1,63 @@
+// Special Tokens
+enum MarkdownTokens {
+    H1 = '# ',
+    H2 = '## ',
+    H3 = '### ',
+    H4 = '#### ',
+    H5 = '##### ',
+    H6 = '###### ',
+}
+
+function isHeadingLevel1(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H1);
+}
+
+function isHeadingLevel2(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H2);
+}
+
+function isHeadingLevel3(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H3);
+}
+
+function isHeadingLevel4(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H4);
+}
+
+function isHeadingLevel5(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H5);
+}
+
+function isHeadingLevel6(line: string): boolean {
+    return line.startsWith(MarkdownTokens.H6);
+}
+
+// Line formatter
+export function formatHtmlForMarkdownLine(line: string) {
```

Comment:
can you use `regex` instead of using the hardcoded `.md` syntax? this way, we can simplify code a lot.

Reply:
@avelynhc I've updated the code to use regular expressions